### ISSUE TITLE
Padding

### ISSUE THREAD

**Issue by Ghastor** [RandomEngy/VidCoder#1178]
### Problem Description

Hi!
There is a big problem with padding option.
The aspect ratio after crop is very wrong!

I need to resize a movie from bluray disk to 1024x576p squared pixels.
My crop values are 202 left and 202 right (black bars)
Non problem with crop values only, but image is wrong when using padding.
Vidcoder fill with 107 left and 107 right pixel to keep 1024x576 image, but the ar letterboxed is wrong, like 4:3 forced letterboxed.

### What version of VidCoder are you running?

9.10 beta

### Encode Log

```text
Not encoded yet
```

**Comment by RandomEngy** [comment]
Can you send me a screenshot of your Sizing tab?

<img width="480" alt="image" src="https://github.com/RandomEngy/VidCoder/assets/9450861/76825e4c-f509-4234-945e-9c86e0b3d46a">

And the Preview window? Does the aspect ratio look wrong in the Preview window, on the final video, or on both?

**Comment by Ghastor** [comment]
On preview windows is wrong.
Ok in final video.
Can you fix it, please?

**Comment by RandomEngy** [comment]
Ahh, I see what happened. When I first added support for padding, the HandBrake API did not include it in the image previews. I needed to add in my own padding emulation in the preview window. But now they put the padding right into the preview image, so it was double-adding the padding there and squishing the preview image. I'll remove the extra layer here and it should fix this. Thanks for reporting.
